modules/apps/object/object-test/src/testFunctional/tests/ObjectRootModel.testcase

```diff
@@ -0,0 +1,151 @@
+@component-name = "portal-object"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Object";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		ObjectAdmin.deleteAllCustomObjectsViaAPI();
+
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-189380 - Verify the user can to see relationships tabs in the initial out-of-the-box layout of one level of children"
+	@priority = 3
+	test CannotViewChildPortlet {
+		task ("Given two custom object definitions, Parent and Child, with a text field and a 1:M relationship between them") {
+			for (var objectName : list "Parent,Child") {
+				ObjectAdmin.addObjectViaAPI(
+					labelName = ${objectName},
+					objectName = ${objectName},
+					pluralLabelName = ${objectName});
+			}
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Text Field",
+				fieldName = "textField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "Parent");
```

Comment:
Since the child object already has a relationship field, we just need to create a field for the parent object